### ISSUE TITLE
Playtime Not Correct  (Multiple Libraries)

### ISSUE THREAD

**Issue by shangles** [JosefNemec/Playnite#929]
Please use search first to see if the issue is already opened. Thank you

**Comment by JosefNemec** [comment]
"Playnite Not Corerrt" doesn't tell me anything, I'll need more information.

**Comment by shangles** [comment]
I just installed Playtime on my Windows 10 Pro 64-bit PC.  I have the following libraries authenticated:

GOG, Origin, Steam, Twitch, UPlay

I used the Export feature to generate a CSV of all of my games and sorted by Playtime.  I noticed several games on Uplay and Origin that reported 0 in the Playtime column. 

Here's a list of games from the CSV that I've played currently or in the past for sure:

Battlefield Hardline Digital Deluxe	Origin	3/20/2015 12:00:00 AM	0	True
Tom Clancy's Rainbow Six Siege	Uplay	12/1/2015 12:00:00 AM	0	True
Battlefield 4	Origin	11/1/2013 12:00:04 AM	0	True
Far Cry 3 Blood Dragon	Uplay	4/30/2013 12:00:00 AM	0	True
Splinter Cell Blacklist	Uplay		0	True
Battlefield V	Origin	11/19/2018 11:00:00 PM	0	True
